pages/common/packtpub.md

```diff
@@ -0,0 +1,32 @@
+# packtpub
+
+> Download freely offered books from packtpub.com.
+> More information: <https://github.com/vladimyr/packtpub-cli>.
+
+- Download the daily offer book into the current directory with the specified book format (defaults to `pdf`):
+
+`packtpub download --type {{pdf|ebup|mobi}}`
+
+- Download the daily offer book into the specified directory:
+
+`packtpub download --dir {{path/to/directory}}`
+
+- Start an interactive login to packtpub.com:
+
+`packtpub login`
+
+- Logout from packtpub.com:
+
+`packtpub logout`
+
+- Display the daily offer:
+
+`packtpub view-offer`
+
+- Open the the daily offer in the default web browser:
```

Comment:
```suggestion
- Open the daily offer in the default web browser:
```

Reply:
Applied change because this has been sitting around a while and it's an obvious typo.

Reply:
Yes, I for some reason forgot to apply it, though I intended to twice!?! 🤷🏻 😄